source/Preferences.cpp

```diff
@@ -48,6 +48,9 @@ namespace {
 	// Enable standard VSync by default.
 	const vector<string> VSYNC_SETTINGS = {"off", "on", "adaptive"};
 	int vsyncIndex = 1;
+
+	const vector<string> AUTO_AIM_SETTINGS = {"off", "when firing", "always on"};
```

Comment:
If the auto aim is "on" currently, then after this PR is merged 1 => "when firing". So "always on" should come before "when firing", so that a user's preference isn't switched from under them.